pkg/resource/table/hooks.go

```diff
@@ -140,6 +144,15 @@ func (rm *resourceManager) customUpdateTable(
 	ko := desired.ko.DeepCopy()
 	rm.setStatusDefaults(ko)
 
+	if delta.DifferentAt("Spec.TimeToLive") {
+		if err := rm.syncTTL(ctx, latest, desired); err != nil {
+			// Ignore "already disabled errors"
+			if awsErr, ok := ackerr.AWSError(err); ok && !(awsErr.Code() == "ValidationException" &&
+				strings.HasPrefix(awsErr.Message(), "TimeToLive is already disabled")) {
```

Comment:
Does a similar error not occur for already enabled?

Reply:
No, it simply updates the `AttributeName` if you set it again